Makefile

```diff
@@ -217,9 +217,9 @@ update-go-deps:
 .PHONY: tidy
 tidy:
 	go mod tidy -v
-	cd pkg/apis/monitoring && go mod tidy -v -modfile=go.mod -compat=1.18
-	cd pkg/client && go mod tidy -v -modfile=go.mod -compat=1.18
-	cd scripts && go mod tidy -v -modfile=go.mod -compat=1.18
+	cd pkg/apis/monitoring && go mod tidy -v -modfile=go.mod
```

Comment:
I have removed `-compat=1.18`, I think this was done couple of years back but looks to be fine without that now? I will wait for CI to confirm